ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml

```diff
@@ -15,7 +15,7 @@ images:
   inputs:
     bin:
       as:
-      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.13
```

Comment:
you need https://github.com/openshift/cluster-policy-controller/blob/master/Dockerfile.rhel#L1 first

Reply:
Fair, I'll do that in a followup to https://github.com/openshift/cluster-policy-controller/pull/16

Reply:
that PR needs to go first, the tests on this PR have no value. you also need to retrigger them when it merges. The replacement is done by string match so if you change the CI config to golang 1.13 first, you are not using it and the mismatch means you have disabled the replacement and it will use the 1.12 from dockerfile.

Reply:
I think I might have misinterpreted the follow up message and you already intended to do the right thing :)